test/distributed/org/apache/cassandra/distributed/impl/IsolatedExecutor.java

```diff
@@ -39,6 +39,7 @@
 import java.util.function.BiFunction;
 import java.util.function.Consumer;
 import java.util.function.Function;
+import java.util.function.Supplier;
```

Comment:
It is dead, but I've used it a few times while developing this patch, and had to work around and use `Function` to emulate the same. I'd leave it since it's quite useful.